bazel/repository_locations.bzl

```diff
@@ -846,8 +846,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ host implementation)",
         project_desc = "WebAssembly for Proxies (C++ host implementation)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-host",
-        version = "6dab125d7a668c7158848b6f48c67fd827c952e6",
-        sha256 = "b5c73ed053a7079bd8bf53b14c4811e87ae521d9fcf4769ec5b248202a27600d",
+        version = "993766996269bd1e41dc625063cfcf4904d940a0",
```

Comment:
Could you update this to the latest? https://github.com/proxy-wasm/proxy-wasm-cpp-host/commit/5a53cf4b231599e1d2a1f2f4598fdfbb727ff948

Reply:
Updated